owncloud/owncloud.py

```diff
@@ -651,6 +667,111 @@ def get_shares(self, path='', **kwargs):
                 return shares
         raise ResponseError(res)
 
+    def create_user_ajax(self, adminuser, adminpass, user_name, initial_password):
```

Comment:
Please remove this, it doesn't use the public API.

`create_user` should do, right ?